css/index.css

```diff
@@ -143,4 +144,179 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+/* ------------Added CSS---------------- */
+/* ------------------------------------- */
+/* ------------------------------------- */
+/* ------------------------------------- */
+
+/*-----------------both html pages whole container-------------------------- */
+
```

Comment:
can do something like 
```
* {
box-sizing:border-box;
}
```
or
``` 
html {
box-sizing: border-box;
}
*{
box-sizing: inherit;
}
```
so you don't need to specify more `box-sizing` later